apps/services/auth/admin-api/src/openApi.ts

```diff
@@ -1,8 +1,27 @@
 import { DocumentBuilder } from '@nestjs/swagger'
+import { environment } from './environments'
+import { AuthScope } from '@island.is/auth/scopes'
 
 export const openApi = new DocumentBuilder()
   .setTitle('IdentityServer Admin api')
   .setDescription('Api for administration.')
   .setVersion('2.0')
   .addTag('auth-admin-api')
+  .addOAuth2(
+    {
+      type: 'oauth2',
+      description:
+        'Authentication and authorization using island.is authentication service (IAS).',
+      flows: {
+        authorizationCode: {
+          authorizationUrl: `${environment.auth.issuer}/connect/authorize`,
+          tokenUrl: `${environment.auth.issuer}/connect/token`,
+          scopes: {
+            openid: 'Default openid scope',
+          },
```

Comment:
This now is technically incorrect as scopes are missing that are authorizing the endpoint. But this is only DX for us and we are testing it with Postman instead of SwaggerUI so doesn't matter.